demo/src/app/default/default.component.html

```diff
@@ -12,25 +13,77 @@
         class="btn btn-outline-primary"
         [routerLink]="['/getting-started']">Installation</a>
     </p>
-    <p>
-      Currently v{{version}}
-    </p>
   </div>
 </main>
 
 <section class="container">
-  <h2>Welcome</h2>
-  <p class="lead">
-     Here we are, the Angular version of the <a target="_blank" href="https://github.com/angular-ui/bootstrap">Angular UI Bootstrap</a>
-     library. This library is being built from scratch in Typescript using the Bootstrap 4 CSS framework.
-  </p>
-  <p class="lead">
-    To get started quickly you can follow <a [routerLink]="['/getting-started']">installation instructions</a> or just browse <a [routerLink]="['/components']">demos</a> for the existing components.
-  </p>
-  <p class="lead">
-    As Bootstrap 4, this library is a work in progress.
-    Please check out our <a href="https://github.com/ng-bootstrap/ng-bootstrap/wiki/Roadmap" target="_blank">roadmap</a> and
-    <a href="https://github.com/ng-bootstrap/ng-bootstrap/issues" target="_blank">list of issues</a> to see all the things we are implementing.
-    Feel free to make comments there.
-  </p>
+
+  <div class="row">
+    <div class="col-2">
+      <img src="/img/cpu.svg">
+    </div>
+    <div class="col-4">
+      <h4>Native</h4>
+      <p class="text-muted">
+        Angular-specific widgets built from ground and using Bootstrap 4 CSS. APIs that makes sense in the Angular ecosystem.
+        No dependencies on any 3rd party JavaScript.
+      </p>
+    </div>
+
+    <div class="col-2">
+      <img src="/img/browser.svg">
+    </div>
+    <div class="col-4">
+      <h4>Widgets</h4>
+      <p class="text-muted">
+        All the Bootstrap widgets (ex. carousel, modal, popovers, tooltips, tabs, ...) and several additional goodies (datepicker, rating, timepicker, typeahead).
+      </p>
+    </div>
+  </div>
+
+  <div class="row mt-5">
+    <div class="col-2">
+      <img src="/img/test.svg">
+    </div>
+    <div class="col-4">
+      <h4>Quality</h4>
+      <p class="text-muted">
+        All code is tested (almost 100% test coverage) and all changes reviewed. We are not cutting corners.
```

Comment:
Might not need `(almost 100% test coverage)`. May be better to link to the actual test coverage or something, where the latest results will be. That way if we ever do hit 100 you don't need to come back and change this.